app/users/urls.py

```diff
@@ -0,0 +1,19 @@
+from django.urls import path
+from rest_framework_simplejwt.views import (
+    TokenObtainPairView,
+    TokenRefreshView,
+)
+
+from app.users.views import PasswordReset, ResetPasswordAPI, UserRegister
+
+urlpatterns = [
+    path("token/", TokenObtainPairView.as_view(), name="token_obtain_pair"),
+    path("token/refresh/", TokenRefreshView.as_view(), name="token_refresh"),
+    path("api/register/", UserRegister.as_view(), name="register"),
```

Comment:
you are repeating api, isn't  /api this in the main urls.py

Reply:
Change made